lib/web_ui/lib/ui_web/src/ui_web/flutter_views_proxy.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'dart:js_interop';
+import 'package:ui/src/engine.dart';
+
+/// The proxy instance for the internal `viewManager` of this app.
+final FlutterViewManagerProxy viewsProxy = FlutterViewManagerProxy();
+
+/// Proxy class for the internal [FlutterViewManager].
+class FlutterViewManagerProxy {
+  /// Returns the `initialData` configuration value passed from JS when `viewId` was added.
+  ///
+  /// Developers can access the initial data from Dart in two ways:
+  ///  * Defining their own `staticInterop` class that describes what you're
+  ///    passing to your views, to retain type safety on Dart (preferred).
+  ///  * Calling [NullableUndefineableJSAnyExtension.dartify] and accessing the
+  ///    returned object as if it were a [Map] (not recommended).
+  JSAny? getInitialData(int viewId) {
```

Comment:
Can this return `Object?` instead of `JSAny?`? The reason is that eventually we'll be able to create `FlutterView` instances from within the framework, i.e. pure Dart code. It would be weird to have to boomerang the data through JS interop. `Object?` would cover both use-cases. The developer knows what kind of data they are passing to the view, and they can cast to the appropriate type.

Reply:
If I can cast Object? to my JS-interop objects: yes, I don't mind. Let me give this a shot.

Reply:
Changed to return `Object?`, since I can cast to `SomethingSomething extends JSObject` in my app later.